### ISSUE TITLE
Test fails on Windows

### ISSUE THREAD

**Issue by jorol** [jorol/Catmandu-MAB2#6]
Test for MABxml importer fails on some Windows system, see [cpantesters](http://matrix.cpantesters.org/?dist=Catmandu-MAB2%200.09;os=mswin32;perl=5.20.1;reports=1).

[Error message](http://www.cpantesters.org/cpan/report/45f45f50-6bf6-1014-9358-33c0c78ce5e0):
```terminal
Read more bytes than requested. Do you use an encoding-related PerlIO layer? at C:\STRAWB~1\cpan\build\Catmandu-MAB2-0.09-smj89Z\blib\lib/MAB2/Parser/XML.pm line 50.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 3.
t/02-importer.t ......... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 3 subtests passed 
```

**Comment by jorol** [comment]
solved e31b6833c578f5cd39eaf58f66f819fc2c6d4c85

**Comment by ElythNet** [comment]
How ?? please help other
